backend/ecs_tasks/delete_files/Dockerfile

```diff
@@ -1,9 +1,9 @@
 ARG src_path=backend/ecs_tasks/delete_files
 ARG layers_path=backend/lambda_layers
 
-FROM python:3.9-slim as base
+FROM public.ecr.aws/docker/library/python:3.9-slim-bullseye as base
```

Comment:
I understand bullseye could be a version available but not declared stable yet, why do we want that?

Reply:
"[bullseye](https://www.debian.org/releases/bullseye/)" refers to the Debian release that the image is built on. It is the current "[stable](https://wiki.debian.org/Status/Stable)" version of Debian.

Reply:
The reason we would want to pin to a particular Debian release is that the stable release has minimal changes in terms of updates.